app/src/main/java/com/everest/moviedb/network/MovieRepository.kt

```diff
@@ -0,0 +1,44 @@
+package com.everest.moviedb.network
+
+import com.everest.moviedb.models.Movie
+import com.everest.moviedb.utils.IMAGE_BASE_URL
+
+
+class MovieRepository(
+    private val retrofitClient: RetrofitClient,
+    private val movieDatabase: MovieDatabase
+) {
+    suspend fun getPopularMovies(): List<Movie> {
+        val movieList = retrofitClient.getClient().getPopularMovies()
```

Comment:
Please handle exception also, if `getPopularMovies` throws an error/network error, we should handle it